portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,34 @@ function addFunFacts() {
     const factsContainer = document.getElementById('facts-container');
     factsContainer.innerText = fact;
 }
+
+
+/** Fetches the current date from the server and adds it to the page. */
```

Comment:
This commented out function should be removed as well.

Reply:
Okay.